src/boot/efi/boot.c

```diff
@@ -1950,6 +1988,32 @@ static VOID config_entry_add_linux(
         uefi_call_wrapper(linux_dir->Close, 1, linux_dir);
 }
 
+static BOOLEAN config_entry_add_signing_keys(
+                Config *config,
+                CHAR16 *name,
+                CHAR16 *keys_dir) {
+        ConfigEntry *entry;
+        CHAR16 *title = NULL, *id = NULL;
+
+        id = PoolPrint(L"signing-keys-%s", name);
```

Comment:
Please use auto- prefix for any automatically created entries.

Reply:
I wonder if "auto-" is the right prefix. So far we used "auto-" for everything we auto-detected not based on stuff the developer explicitly configured. but theys keys are placed there by someone, so they aren't really "auto" i'd say, the names are conciously chosen by the developer, not by our program or us. 

hence i think the approach @vdagonneau-anssi took makes a lot of sense to me: prefix it with some string that makes it unlikely we clash with regular menu items, and allows us to discern them from regular menu items, but not make it "auto".

of course, strictly speaking this is kind of an api breakage, because if people actually already had menu items named in that format things will break... but I'd argue the risk of that is kinda small, the prefix chosen is long enough to make this unlikely enough.

That said, it would probably make sense to skip over any regular menu items we find in the dir if they begin with "auto-" or "signing-keys-", just in case, to avoid confusion.

Reply:
here too, we should probably settle on calling these "signature-keys", hence change to `signature-keys-%s` here too, just to make this systematic, and avoid needless discrepancies in nomenclature between human facing naming and code